review/tests/test_review.py

```diff
@@ -1,22 +1,56 @@
 import pytest
+import datetime
 from datetime import timedelta
 from django.utils import timezone
 from review.models import Review
 from django.template.backends import django
 
+from django.contrib.auth.models import User
+from account.models.professional import Professions, Professional
+from account.models.profile import Profile, UserType
+from account.models.client import Client
+
+CLIENT_USERNAME = 'ClientEntity'
+PROFESSIONAL_USERNAME = 'ProfessionalEntity'
+RATING = '4'
+DESCRIPTION = 'Creating a test review'
+
 
 @pytest.fixture
-def review():
-    now = timezone.now()
-    review = Review.objects.create(rating='4', description='Creating a test review...',
-                                   date_posted=now - timedelta(days=50),
-                                   client_id='client_id', professional_id='professional_id')
-    return review
+def create_review():
+    def _create_review():
+        # Client instance
+        client_user = User.objects.create_user(username=CLIENT_USERNAME, password='password',
+                                               first_name='Client', last_name='Entity', email='client@entity.com',
+                                               last_login=datetime.datetime.now(datetime.timezone.utc))
+        client_profile = Profile(user_id=client_user, user_type=UserType.Client,
+                                 phone_number='1010101010', country='Israel', city='Tel Aviv', address='Address')
+        client_profile.save()
+        client = Client(profile_id=client_profile, birthday=datetime.date(2000, 1, 1))
+        client.save()
+        # Professional instance
+        professional_user = User.objects.create_user(
+            username=PROFESSIONAL_USERNAME, password='password',
+            first_name='Professional', last_name='Entity', email='professional@entity.com',
+            last_login=datetime.datetime.now(datetime.timezone.utc))
+        professional_profile = Profile(user_id=professional_user, user_type=UserType.Professional,
+                                       phone_number='0101010101', country='Israel', city='Tel Aviv', address='Address')
+        professional_profile.save()
+        professional = Professional(profile_id=professional_profile,
```

Comment:
Why don't you use the profiles prepared by Tal and found in account/test.py

Reply:
Done

Reply:
> Why don't you use the profiles prepared by Tal and found in account/test.py

Done